.circleci/config.yml

```diff
@@ -229,7 +229,6 @@ jobs:
     docker:
       - image: quay.io/influxdb/influxdb-circleci:latest
     working_directory: /go/src/github.com/influxdata/influxdb
-    parallelism: 8
```

Comment:
Our JS tests are so quick, there's not much point to parallelizing across many VMs